### PR TITLE
outline-label-mediumlarge-cy-169

### PR DESCRIPTION
**Pull request by cris22380**

### CODE REVIEW

File: src/components/out-line-label/OutLineLabel.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+interface IOutLineLabel {
+  icon?: ReactNode,
+  image?: ReactNode,
+  colorLabel?: ReactNode,
+  badgeLabel?: ReactNode,
+  activated?: boolean,
+  size?: 'medium' | 'large',
+}
+
+const OutLineLabel: FC<IOutLineLabel> = props => {
```
Comment (diegomart2000) [inline src/components/out-line-label/OutLineLabel.tsx]: use props destructuring.

File: src/components/out-line-label/OutLineLabel.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  return (
+    <SOutLineLabel {...props} >
+      {props.colorLabel && <SColorLabelWrapper {...props}>{props.colorLabel}</SColorLabelWrapper>}
```
Comment (diegomart2000) [inline src/components/out-line-label/OutLineLabel.tsx]: Only pass down required properties.

File: src/components/out-line-label/styles/SIconWrapper.tsx [hunk 3]
```diff
@@ -0,0 +1,36 @@
+import styled, { css } from 'styled-components';
+import { MIN_SILVER, MIN_GOLD, MIN_PLATINUM, MIN_DIAMOND } from '../../../constants/styles/mediaquerys';
+
+interface ISIconWrapper {
+  size?: 'medium' | 'large';
```
Comment (diegomart2000) [inline src/components/out-line-label/styles/SIconWrapper.tsx:5]: This size interface is going to be a global type. Will let you know.

**Review by diegomart2000 — CHANGES_REQUESTED** [review]

**Comment by cris22380** [comment]
Screen Shot 2021-12-06 at 15 29 13
Screen Shot 2021-12-06 at 15 29 24
Screen Shot 2021-12-06 at 15 29 43
Screen Shot 2021-12-06 at 15 30 57
Screen Shot 2021-12-06 at 15 34 21
Screen Shot 2021-12-06 at 15 34 37
Screen Shot 2021-12-06 at 15 34 54
Screen Shot 2021-12-06 at 15 35 44
Screen Shot 2021-12-06 at 15 40 20
Screen Shot 2021-12-06 at 15 40 32
Screen Shot 2021-12-06 at 15 40 40
Screen Shot 2021-12-06 at 15 41 25
Screen Shot 2021-12-06 at 15 41 59
Screen Shot 2021-12-06 at 15 42 07
Screen Shot 2021-12-06 at 15 42 16
Screen Shot 2021-12-06 at 15 42 49
Screen Shot 2021-12-06 at 15 45 20
Screen Shot 2021-12-06 at 15 45 39
Screen Shot 2021-12-06 at 15 45 57
Screen Shot 2021-12-06 at 15 46 27
Screen Shot 2021-12-06 at 15 47 37
Screen Shot 2021-12-06 at 15 47 58
Screen Shot 2021-12-06 at 15 48 07
Screen Shot 2021-12-06 at 15 48 33
Screen Shot 2021-12-06 at 15 48 39
Screen Shot 2021-12-06 at 15 49 13
Screen Shot 2021-12-06 at 15 49 22
Screen Shot 2021-12-06 at 15 49 34
Screen Shot 2021-12-06 at 15 49 59
Screen Shot 2021-12-06 at 15 50 05
Screen Shot 2021-12-06 at 15 52 18
Screen Shot 2021-12-06 at 15 52 29
Screen Shot 2021-12-06 at 15 52 40
Screen Shot 2021-12-06 at 15 53 13
Screen Shot 2021-12-06 at 15 53 24
Screen Shot 2021-12-06 at 15 55 13
Screen Shot 2021-12-06 at 15 55 23
Screen Shot 2021-12-06 at 15 55 31
Screen Shot 2021-12-06 at 15 56 00
Screen Shot 2021-12-06 at 15 56 06

**Review by diegomart2000 — APPROVED** [review]

**Pull request merged**